diff --git a/debian/patches/01_config_file_relocation.patch b/debian/patches/01_config_file_relocation.patch
old mode 100644
new mode 100755
diff --git a/src/file.c b/src/file.c
index 46d2c65..c51c97b 100644
--- a/src/file.c
+++ b/src/file.c
@@ -179,6 +179,11 @@ char *file_findControlFile() {
     return (rcfile);
   }
   memset(rcfile, 0, STRLEN);
+  snprintf(rcfile, STRLEN, "/etc/monit/%s", MONITRC);
+  if(file_exist(rcfile)) {
+    return (rcfile);
+  }
+  memset(rcfile, 0, STRLEN);
   snprintf(rcfile, STRLEN, "/etc/%s", MONITRC);
   if(file_exist(rcfile)) {
     return (rcfile);
@@ -199,8 +204,8 @@ char *file_findControlFile() {
     return (rcfile);
   }
   LogError("%s: Cannot find the control file at "
-      "~/.%s, /etc/%s, %s/%s, /usr/local/etc/%s or at ./%s \n",
-      prog, MONITRC, MONITRC, SYSCONFDIR, MONITRC, MONITRC, MONITRC);
+       "~/.%s, /etc/monit/%s, /etc/%s, /usr/local/etc/%s or at ./%s \n",
+       prog, MONITRC, MONITRC, MONITRC, MONITRC, MONITRC);
   exit(1);
   
 }
