diff --git a/monitrc b/monitrc
index 7b31576..76bf691 100644
--- a/monitrc
+++ b/monitrc
@@ -25,14 +25,14 @@ set daemon  120             # check services at 2-minute intervals
 ## omitted, Monit will use 'user' facility by default. If you want to log to 
 ## a standalone log file instead, specify the full path to the log file
 #
-# set logfile syslog facility log_daemon                       
+set logfile /var/log/monit.log
 #
 #
 ### Set the location of the Monit id file which stores the unique id for the
 ### Monit instance. The id is generated and stored on first Monit start. By 
 ### default the file is placed in $HOME/.monit.id.
 #
-# set idfile /var/.monit.id
+set idfile /var/lib/monit/id
 #
 ### Set the location of the Monit state file which saves monitoring states
 ### on each cycle. By default the file is placed in $HOME/.monit.state. If
@@ -40,7 +40,7 @@ set daemon  120             # check services at 2-minute intervals
 ### the monitoring state across reboots. If it is on temporary filesystem, the
 ### state will be lost on reboot which may be convenient in some situations.
 #
-# set statefile /var/.monit.state
+set statefile /var/lib/monit/state
 #
 ## Set the list of mail servers for alert delivery. Multiple servers may be 
 ## specified using a comma separator. If the first mail server fails, Monit 
@@ -59,9 +59,9 @@ set daemon  120             # check services at 2-minute intervals
 ## size using the SLOTS option (if omitted, the queue is limited by space 
 ## available in the back end filesystem).
 #
-# set eventqueue
-#     basedir /var/monit  # set the base directory where events will be stored
-#     slots 100           # optionally limit the queue size
+set eventqueue
+    basedir /var/lib/monit/events # set the base directory where events will be stored
+    slots 100                     # optionally limit the queue size
 #
 #
 ## Send status and events to M/Monit (for more informations about M/Monit 
@@ -244,6 +244,6 @@ set httpd port 2812 and
 ## It is possible to include additional configuration parts from other files or
 ## directories.
 #
-#  include /etc/monit.d/*
+include /etc/monit/conf.d/*
 #
 #
