From c2d6eb5419da9acdaa184b0a34476b6e2b3c8dfb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9bastien=20Gross?= <seb•ɑƬ•chezwam•ɖɵʈ•org>
Date: Wed, 15 Jun 2011 08:23:03 +0000
Subject: [PATCH 5/5] Applying 07_fqdn_in_HOST.patch
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Sébastien Gross <seb•ɑƬ•chezwam•ɖɵʈ•org>
---
 p.y    |    2 +-
 util.c |   25 +++++++++++++++++++++++++
 util.h |    7 +++++++
 3 files changed, 33 insertions(+), 1 deletions(-)

diff --git a/p.y b/p.y
index 757bf50..eea3aaf 100644
--- a/p.y
+++ b/p.y
@@ -1866,7 +1866,7 @@ static void preparse() {
   /*
    * Get the localhost name
    */
-  if (Util_getfqdnhostname(localhost, sizeof(localhost)))
+  if (Util_getfqdnhostname(localhost, sizeof(localhost)) < 0)
     snprintf(localhost, STRLEN, "%s", LOCALHOST);
 
   /* Set instance incarnation ID */
diff --git a/util.c b/util.c
index 4f50e7b..d6ef9e0 100644
--- a/util.c
+++ b/util.c
@@ -100,6 +100,10 @@
 #include <crypt.h>
 #endif
 
+#ifdef HAVE_NETDB_H
+#include <netdb.h>
+#endif
+
 #ifdef HAVE_NETINET_IN_H
 #include <netinet/in.h>
 #endif
diff --git a/util.h b/util.h
index a6e7ba4..5e5fd15 100644
--- a/util.h
+++ b/util.h
@@ -515,6 +515,13 @@ char *Util_portDescription(Port_T p, char *buf, int bufsize);
  */
 void Util_stringbuffer(Buffer_T *b, const char *m, ...);
 
+/**
+ *  Returns the FQDN hostname or fallback to gethostname() output
+ *  @param buf the character array for hostname
+ *  @param len the number of bytes in buf
+ *  @return On success, zero is returned.  On error, -1 or -2.
+ */
+int Util_getfqdnhostname(char* buf, unsigned len);
 
 /**
  *  Returns the FQDN hostname or fallback to gethostname() output
-- 
1.7.2.3

