diff --git a/src/p.y b/src/p.y
index d718c7e..bc1e500 100644
--- a/src/p.y
+++ b/src/p.y
@@ -1861,7 +1861,7 @@ static void preparse() {
   /*
    * Get the localhost name
    */
-  if (Util_getfqdnhostname(localhost, sizeof(localhost)))
+  if (Util_getfqdnhostname(localhost, sizeof(localhost)) < 0)
     snprintf(localhost, STRLEN, "%s", LOCALHOST);
 
   /* Set instance incarnation ID */
diff --git a/src/util.c b/src/util.c
index 023f5a9..342148e 100644
--- a/src/util.c
+++ b/src/util.c
@@ -95,6 +95,10 @@
 #include <crypt.h>
 #endif
 
+#ifdef HAVE_NETDB_H
+#include <netdb.h>
+#endif
+
 #ifdef HAVE_NETINET_IN_H
 #include <netinet/in.h>
 #endif
diff --git a/src/util.h b/src/util.h
index 16eea68..f12a8c2 100644
--- a/src/util.h
+++ b/src/util.h
@@ -510,6 +510,13 @@ char *Util_portDescription(Port_T p, char *buf, int bufsize);
  */
 void Util_stringbuffer(Buffer_T *b, const char *m, ...);
 
+/**
+ *  Returns the FQDN hostname or fallback to gethostname() output
+ *  @param buf the character array for hostname
+ *  @param len the number of bytes in buf
+ *  @return On success, zero is returned.  On error, -1 or -2.
+ */
+int Util_getfqdnhostname(char* buf, unsigned len);
 
 /**
  *  Returns the FQDN hostname or fallback to gethostname() output
